file_paths.md

```diff
@@ -275,6 +279,12 @@ create a `puppet` user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        bin *
+            pxp-agent.exe *
+        pxp-modules *
+            pxp-module-puppet *
```

Comment:
I like naming the script `pxp-module-puppet` (for ease of searching for the file and for being descriptive) and the top directory of `pxp-agent` makes sense to me. I'm pretty agnostic about the intermediate directory. I.e. either of these seem fine to me:
* `pxp-agent\pxp-modules\pxp-module-puppet`
* `pxp-agent\modules\pxp-module-puppet`

Coin-flip there for me. @Iristyle: preferences? or other suggestions?

Whatever we settle on, let's be consistent across the windows and *nix paths.